ydb/core/ydb_convert/table_description.cpp

```diff
@@ -780,6 +784,38 @@ bool FillColumnDescription(NKikimrSchemeOp::TAlterColumnTable& out, const google
     return FillColumnDescriptionImpl(out, in, status, error);
 }
 
+bool FillColumnFamily(
+    const Ydb::Table::ColumnFamily& from, NKikimrSchemeOp::TFamilyDescription* to, Ydb::StatusIds::StatusCode& status, TString& error) {
+    to->SetName(from.name());
+    if (from.has_data()) {
+        status = Ydb::StatusIds::BAD_REQUEST;
+        error = "Field `DATA` is not supported for OLAP tables";
```

Comment:
This is the message that will be returned to the user.
So accuracy matters.

I would suggest adding column family name, the same way its done below.